py/objint_longlong.c

```diff
@@ -43,6 +44,10 @@
 const mp_obj_int_t mp_sys_maxsize_obj = {{&mp_type_int}, MP_SSIZE_MAX};
 #endif
 
+static void raise_long_long_overflow(void) {
+    mp_raise_msg(&mp_type_OverflowError, MP_ERROR_TEXT("result overflows signed 64-bit"));
```

Comment:
`long long` is at least 64-bits, but could possibly be larger (is that every true in practice? not sure).  So maybe the message can be "result overflows long long storage"?

Reply:
Ah C, so many little pitfalls... (Changed.)